Applications/Utils/ModelPreparation/PartitionMesh/PartitionMesh.cpp

```diff
@@ -0,0 +1,146 @@
+/*!
+  \file PartitionMesh.cpp
+  \date   2016.05
+
+  \brief  A tool for mesh partitioning.
+
+  \copyright
+  Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+             Distributed under a Modified BSD License.
+               See accompanying file LICENSE.txt or
+               http://www.opengeosys.org/project/license
+
+*/
+
+#include <tclap/CmdLine.h>
+
+#ifdef _MSC_VER
+#include <windows.h>
```

Comment:
Is it correct that windows.h is only included when using VisualStudio compiler? What about gcc under windows os?